src/pages/STG_PanelCreatePayments.page

```diff
@@ -1,21 +1,45 @@
 <apex:page controller="STG_PanelCreatePayments_CTRL" docType="html-5.0" standardStylesheets="false" >
+    <apex:includeLightning/>
+    <script>
```

Comment:
I know this is a `<script>` block, but I'm wondering if we can put this little block into a VF component that all the individual pages load as
`<c:STG_PermissionsCheckAndNotify hasAccess={hasAccess} />`.

If this works, then any other STG page using the same block should be updated to use this (even if that page wasn't changed in this PR).

Reply:
I am unable to create a vf component for to load the lightning out. From the error it seem like the component will be loaded on the initial with `STG_Manager` and the error saying Lightning-out is already hooked to something else will be thrown.

Reply:
Thanks for checking into it.